app.ts

```diff
@@ -33,14 +35,10 @@ export type Secretary = {
   title: string;
 };
 
-function generateMinutesName(meeting: Meeting): string {
-  const padZeroes = (n: number) => String(n).padStart(2, '0');
-
-  const { plannedStart } = meeting;
-  const year = plannedStart.getFullYear();
-  const month = padZeroes(plannedStart.getMonth() + 1);
-  const day = padZeroes(plannedStart.getDate());
-  return `Notulen - P${year}-${month}-${day}.pdf`.replace('/', '-');
+function generateMinutesFileName(meeting: Meeting): string {
+  const suffix = meeting.kind === MEETING_KINDS.PVV ? '-VV' : '';
+  const name = `VR PV ${meeting.numberRepresentation}${suffix}`;
+  return `${name}.pdf`.replace('/', '-');
```

Comment:
I'm inclined to change this like in decision-report-generation and just use the piece name so as to not have issues with BIS/TER and such.